core-web/libs/dotcms-scss/angular/dotcms-theme/components/_button.scss

```diff
@@ -2,708 +2,278 @@
-    &:enabled:focus {
-        color: $color-palette-primary;
+    &.p-button-sm {
+        border: 1px solid $color-palette-primary;
     }
 
-    .p-button-icon {
-        margin: 0 0 math.div($spacing-1, 2);
-        font-size: $font-size-xl;
+    .p-button-label,
+    .p-button-icon,
+    .pi {
+        color: $color-palette-primary;
     }
-}
 
-.p-button:focus {
-    outline: 0 none;
-    outline-offset: 0;
-    box-shadow: $button-shadow-focus;
-}
-
-.p-button .p-button-icon-left {
-    margin-right: $spacing-1;
-}
-
-.p-button .p-button-icon-right {
-    margin-left: $spacing-1;
-}
-
-.p-button .p-button-icon-bottom {
-    margin-top: $spacing-1;
-}
-
-.p-button .p-button-icon-top {
-    margin-bottom: $spacing-1;
-}
+    &:hover {
+        background-color: $color-palette-primary-100;
+    }
+    &:active {
+        background-color: $color-palette-primary-200;
+    }
+    &:focus {
+        background-color: transparent;
+        box-shadow: 0px 0px 0px 2.8px rgba(18, 67, 227, 0.2); // Ask what should we do
+    }
 
-.p-button .p-badge {
-    margin-left: $spacing-1;
-    min-width: $spacing-3;
-    height: $spacing-3;
-    line-height: $spacing-3;
-    color: $color-palette-primary;
-    background-color: $white;
-}
+    &.p-button-secondary {
+        background-color: transparent;
+        border: 1.5px solid $color-palette-secondary;
+        color: $color-palette-secondary;
 
-.p-button.p-button-raised {
-    box-shadow: 0 3px 1px -2px $color-palette-black-op-20, 0 2px 2px 0 $color-palette-black-op-10,
-        0 1px 5px 0 $color-palette-black-op-10;
-}
-
-.p-button.p-button-rounded {
-    border-radius: $spacing-5;
-}
+        &.p-button-sm {
+            border: 1px solid $color-palette-secondary;
+        }
 
-.p-button-rounded.p-button.p-button-icon-only {
-    color: $color-palette-gray-800;
-    padding: 0.571rem;
-    min-width: auto;
+        .p-button-label,
+        .p-button-icon,
+        .pi {
+            color: $color-palette-secondary;
+        }
 
-    &:enabled {
         &:hover {
-            color: $color-palette-gray-800;
+            background-color: $color-palette-secondary-100;
+        }
+        &:active {
+            background-color: $color-palette-secondary-200;
         }
-
-        &:active,
         &:focus {
-            color: $color-palette-primary;
-            background: $focus-bg;
+            background-color: transparent;
+            box-shadow: 0px 0px 0px 2.8px rgba(18, 67, 227, 0.2); // Ask what should we do
         }
     }
 }
 
-.p-button.p-button-icon-only {
-    padding-left: $spacing-2;
-    padding-right: $spacing-2;
-}
+// Severity for text button
+.p-button-text:enabled {
+    background-color: transparent;
+    color: $black;
+    overflow: hidden;
+    max-width: 100%;
 
-.p-button.p-button-icon-only.p-button-text.p-button-sm {
-    height: $spacing-5;
-    width: $spacing-5;
-    gap: 0;
+    .p-button-label {
+        color: $black;
+        @include truncate-text;
```

Comment:
Why? do we have to added to all the buttons?

Reply:
We could, but for now this behavior is only needed for text buttons that are being used in small places, like content type fields